backend/src/emails/sendChatMessageMail.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { sendChatMessageMail } from './sendEmail'
+
+const senderUser = {
```

Comment:
why not use neode/factory here?

Reply:
Because I do not want to touch the database.